middleware/gateway/unittest/source/test_manager.cpp

```diff
@@ -618,7 +618,7 @@ namespace casual
 
          // because of some unknown factor, the queue-manager needs some time to get ready
          // to handle calls. Strange but is a separate issue.
-         process::sleep( std::chrono::milliseconds{ 10});
+         process::sleep( std::chrono::milliseconds{ 500});
```

Comment:
Is this similar to the issue where some unittests had services being called before they were advertised that we dealt with a few months back?

Reply:
Yes, sort of the same thing.